tests/NPSP.py

```diff
@@ -409,17 +409,14 @@ def select_eng_plan_checkbox(self,title):
             locator=npsp_lex_locators['engagement_plan']['checkbox'].format("label",title)
             self.selenium.get_webelement(locator).click()
             
-    def enter_eng_plan_values(self, **kwargs):
+    def enter_eng_plan_values(self, name, value):
         """Enter values into corresponding fields in Engagement Plan Templet page"""
-        for name, value in kwargs.items():
-            if name == "Engagement Plan Template Name":
-                id = "idName"
-                locator = npsp_lex_locators['id'].format(id) 
-                self.selenium.get_webelement(locator).send_keys(value)     
-            elif name == "Description":
-                id = "idDesc"
-                locator = npsp_lex_locators['id'].format(id)
-                self.selenium.get_webelement(locator).send_keys(value)  
+        locator = npsp_lex_locators['id'].format(name) 
+        self.selenium.set_focus_to_element(locator)
+        field=self.selenium.get_webelement(locator)
+        field.clear()
+        field.send_keys(value)     
```

Comment:
If you add a property to get the Salesforce library at the top of the NPSP class:

    @property
    def salesforce(self):
        return self.builtin.get_library_instance('cumulusci.robotframework.Salesforce')

Then you can reuse the code for populating a field here:

    self.salesforce._populate_field(locator, value)

That's probably better in case we change the best practice for filling in fields. For example I'm in the middle of updating how we clear the existing field value so that it works more reliably.

Reply:
@davisagli thank you. Made these changes